src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/Metadata/IModelBindingMessageProvider.cs

```diff
@@ -30,5 +30,28 @@ public interface IModelBindingMessageProvider
         /// </summary>
         /// <value>Default <see cref="string"/> is "The value '{0}' is invalid.".</value>
         Func<string, string> ValueMustNotBeNullAccessor { get; }
+
+        /// <summary>
+        /// Fallback error message HTML and tag helpers display when a property is invalid but the
+        /// <see cref="ModelError"/>s have <c>null</c> <see cref="ModelError.ErrorMessage"/>s.
+        /// </summary>
+        /// <value>Default <see cref="string"/> is "The value '{0}' is invalid.".</value>
+        Func<string, string> ValueInvalid_UnknownErrorResource { get; }
```

Comment:
Follow the naming conventions of the other ones. Drop the underscore, end with 'Accessor' instead of 'Resource'